hnn_core/network.py

```diff
@@ -438,10 +438,10 @@ def __eq__(self, other):
                                           target_gids=target_gids,
                                           loc=conn['loc'],
                                           receptor=conn['receptor'])
-            if len(match_conns) == 1:
-                # check if connections are equal
-                if conn != other.connectivity[match_conns[0]]:
+            if len(match_conns) >= 1:
+                if not any([conn == other.connectivity[match] for match in match_conns]):
```

Comment:
@ntolley Just realized the edge case we were trying to think of. 
```
net1.connectivity = [c1, c1, c2]
net2.connectivity = [c1, c2, c2]
```
The current logic wouldn't catch cases with this type of duplication since they are of equal lengths and each connection would find at least one match in the other set. Is this worth catching?

Reply:
I don't have an immediate answer right now but this discussion provides some good solutions to the problem:
https://stackoverflow.com/questions/7828867/how-to-efficiently-compare-two-unordered-lists-not-sets?noredirect=1&lq=1

Reply:
Hmmm can test out if that counter method works! Thanks!

Reply:
I updated the eq function with the 3rd option from that thread. Now it will catch those edge cases.